bom-2.332.x/pom.xml

```diff
@@ -124,11 +119,6 @@
                 <artifactId>config-file-provider</artifactId>
                 <version>3.10.0</version>
             </dependency>
-            <dependency>
-                <groupId>org.jenkins-ci.plugins</groupId>
-                <artifactId>credentials</artifactId>
-                <version>1087.1089.v2f1b_9a_b_040e4</version>
```

Comment:
Hmm, pinned in #1065 but why?

I guess fine if CI passes.

Reply:
I've since learned that it was pinned at this version because that is the newest version supported on the 2.332.x line.  This will need to remain.